README.md

```diff
@@ -5,172 +5,46 @@
+Or manually clone the repository at [github.com/DataDog/php-datadogstatsd](https://github.com/DataDog/php-datadogstatsd) and set it up with `require './src/DogStatsd.php'.`
 
-Clone repository at [github.com/DataDog/php-datadogstatsd](https://github.com/DataDog/php-datadogstatsd)
+Once installed, turn on the socket extension to PHP which must be enabled at compile time by giving the `--enable-sockets` option to **configure**.
 
-Setup: `require './src/DogStatsd.php';`
-
-## Usage
-  
-Once installed, turn on the socket extension to PHP which must be enabled at compile time by giving the `--enable-sockets` option to <b>configure</b>.  
-  
-### instantiation
+### Configuration
 
 To instantiate a DogStatsd object using `composer`:
 
 ```php
+<?php
+
 require __DIR__ . '/vendor/autoload.php';
 
 use DataDog\DogStatsd;
-use DataDog\BatchedDogStatsd;
-
-$statsd = new DogStatsd();
-$statsd = new BatchedDogStatsd();
-```
-
-DogStatsd constructor, takes a configuration array. The configuration can take any of the following values (all optional):
-
-- `host`: the host of your DogStatsd server, if not set tries to get it from the `DD_AGENT_HOST` environment variable, default to `localhost`
-- `port`: the port of your DogStatsd server, if not set tries to get it from the `DD_DOGSTATSD_PORT` environment variable, default to `8125`
-- `socket_path`: the path to the DogStatsd UNIX socket (overrides `host` and `port`, only supported with `datadog-agent` >= 6). default to `null`
-- `global_tags`: tags to apply to all metrics sent, the `dd.internal.entity_id` tag is appended to `global_tags` from the `DD_ENTITY_ID` environment variable
-
-When sending `events` over TCP the following options can be set (see [Events section](#submitting-events)):
-
-- `api_key`: needed to send `event` over TCP
-- `app_key`: needed to send `event` over TCP
-- `curl_ssl_verify_host`: Config pass-through for `CURLOPT_SSL_VERIFYHOST` defaults 2
-- `curl_ssl_verify_peer`: Config pass-through for `CURLOPT_SSL_VERIFYPEER` default 1
-- `datadog_host`: where to send events default `https://app.datadoghq.com`
-
-### Tags
-
-The 'tags' argument can be a array or a string. Value can be set to `null`.
-
-```php
-# Both call will send the "app:php1" and "beta" tags.
-$statsd->increment('your.data.point', 1, array('app' => 'php1', 'beta' => null));
-$statsd->increment('your.data.point', 1, "app:php1,beta");
-```
-
-### Increment
-
-To increment things:
-
-``` php
-$statsd->increment('your.data.point');
-$statsd->increment('your.data.point', .5);
-$statsd->increment('your.data.point', 1, array('tagname' => 'value'));
-```
-
-### Decrement
-
-To decrement things:
-
-``` php
-$statsd->decrement('your.data.point');
-```
-
-### Timing
-
-To time things:
-
-``` php
-$start_time = microtime(true);
-run_function();
-$statsd->microtiming('your.data.point', microtime(true) - $start_time);
-
-$statsd->microtiming('your.data.point', microtime(true) - $start_time, 1, array('tagname' => 'value'));
-```
```

Comment:
`microtiming` is not explained in the doc website. We should keep it here.

Reply:
indeed it's not explained, creating a card to add it here: https://docs.datadoghq.com/developers/metrics/dogstatsd_metrics_submission/?tab=php#timer

Reply:
https://github.com/DataDog/documentation/pull/5953